properties-files/openshiftv4_on_azure.properties

```diff
@@ -0,0 +1,53 @@
+ORCHESTRATION_USER=root
+ORCHESTRATION_HOST=
+SSHKEY_TOKEN=
+OPENSHIFT_CLEANUP=false
+OPENSHIFT_INSTALL=true
+OPENSHIFT_POST_INSTALL=true
+OPENSHIFT_POST_CONFIG=true
+OPENSHIFT_DEBUG_CONFIG=false
+OPENSHIFT_CLIENT_LOCATION=
+OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=
+OPENSHIFT_INSTALL_APIVERSION=v1
+OPENSHIFT_INSTALL_SSH_PUB_KEY_FILE=/root/.ssh/id_rsa.pub
+OPENSHIFT_INSTALL_PULL_SECRET=
+OPENSHIFT_INSTALL_QUAY_REGISTRY_TOKEN=
+OPENSHIFT_INSTALL_IMAGE_REGISTRY=
+OPENSHIFT_INSTALL_REGISTRY_TOKEN=
+OPENSHIFT_INSTALL_INSTALLER_FROM_SOURCE=false
+OPENSHIFT_INSTALL_INSTALLER_FROM_SOURCE_VERSION=
+GOPATH=/root/.go
+AZURE_SUBSCRIPTION_ID=
+AZURE_TENANT_ID=
+AZURE_SERVICE_PRINCIPAL_CLIENT_ID=
+AZURE_SERVICE_PRINCIPAL_CLIENT_SECRET=
+AZURE_BASE_DOMAIN_RESOURCE_GROUP_NAME=
+AZURE_REGION="centralus"
+OPENSHIFT_BASE_DOMAIN=
+OPENSHIFT_CLUSTER_NAME=
+OPENSHIFT_MASTER_COUNT=3
+OPENSHIFT_WORKER_COUNT=5
+OPENSHIFT_MASTER_VM_SIZE=Standard_D4s_v3
+OPENSHIFT_WORKER_VM_SIZE=Standard_D4s_v3
+OPENSHIFT_MASTER_ROOT_VOLUME_SIZE=64
+OPENSHIFT_WORKER_ROOT_VOLUME_SIZE=64
+OPENSHIFT_CIDR=10.128.0.0/14
+OPENSHIFT_MACHINE_CIDR=10.0.0.0/16
+OPENSHIFT_SERVICE_NETWORK=172.30.0.0/16
+OPENSHIFT_HOST_PREFIX=23
+OPENSHIFT_POST_INSTALL_POLL_ATTEMPTS=600
+OPENSHIFT_TOGGLE_INFRA_NODE=true
+OPENSHIFT_TOGGLE_WORKLOAD_NODE=true
+MACHINESET_METADATA_LABEL_PREFIX=machine.openshift.io
+OPENSHIFT_INFRA_NODE_INSTANCE_TYPE=m5.large
+OPENSHIFT_WORKLOAD_NODE_INSTANCE_TYPE=Standard_D4s_v3
```

Comment:
Lines 42-43 should change to:

```
OPENSHIFT_INFRA_NODE_VM_SIZE=Standard_D2s_v3
OPENSHIFT_WORKLOAD_NODE_VM_SIZE=Standard_D2s_v3
```

Once https://github.com/openshift-scale/scale-ci-deploy/pull/30 is merged